Task 1/Assets/Scripts/GameController.cs

```diff
@@ -0,0 +1,77 @@
+using UnityEngine;
+
+public class GameController : MonoBehaviour
+{
+    public Vector3 initialSize;
+    public Vector3 initialPosition;
+    public Vector3 startingBoxPosition;
+    public Vector3 startMovingPosition = new Vector3(-1.5f, 0, 0);
+
+    private GameLogic logic;
+    private Camera cam;
+    private Vector3 cameraStartPos;
+    public int Score { get; private set; }
+    public int MaxScore { get; private set; }
+
+    private bool isStarting;
+
+    private void Start()
+    {
+        logic = GetComponent<GameLogic>();
+        logic.Init(initialSize,initialPosition);
+
+        var posY = Mathf.Abs(startingBoxPosition.y*2 + initialSize.y / 2);
```

Comment:
can you please explain why you are using y*2?

Reply:
code styles please - be consistent with spaces:
```suggestion
        var posY = Mathf.Abs(startingBoxPosition.y * 2 + initialSize.y / 2);
```